src/huggingface_hub/repository.py

```diff
@@ -631,6 +631,9 @@ def clone_from(self, repo_url: str, use_auth_token: Union[bool, str, None] = Non
                 repo_url += REPO_TYPES_URL_PREFIXES[self.repo_type]
 
             if token is not None:
+                if token.startswith("api_org"):
+                    raise ValueError("You must use your personal account token.")
```

Comment:
```suggestion
                    raise ValueError("You must use your personal access token.")
```
(it seems the "personal account token" naming is used elsewhere, but it should be user/personal access token, see https://huggingface.co/settings/tokens)

Reply:
maybe say ", not an Organization token" and link to https://huggingface.co/settings/tokens